modules/builds-create-custom-build-artifacts.adoc

```diff
@@ -31,8 +30,8 @@ ENTRYPOINT ["/usr/bin/build.sh"]
 +
 [source,terminal]
 ----
-FROM registry.centos.org/centos:7
-RUN touch /tmp/built
+FROM registry.redhat.io/ubi8/ubi
```

Comment:
The custom build does not work to pull this image, we need add secret here. @adambkaplan 

`oc start-build sample-custom-build -F`
`build.build.openshift.io/sample-custom-build-1 started`
`STEP 1: FROM registry.redhat.io/ubi8/ubi`
`error creating build container: Error initializing source docker://registry.redhat.io/ubi8/ubi:latest: unable to retrieve auth token: invalid username/password: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
`
`error pushing image "image-registry.openshift-image-registry.svc:5000/xiuwang/sample-custom:latest" to "docker://image-registry.openshift-image-registry.svc:5000/xiuwang/sample-custom:latest": error locating image with name "image-registry.openshift-image-registry.svc:5000/xiuwang/sample-custom:latest" ([image-registry.openshift-image-registry.svc:5000/xiuwang/sample-custom:latest]): image not known
`

Reply:
@SNiemann15 to simplify matters, we should use the regular container registry (instead of the terms based registry):

```suggestion
FROM registry.access.redhat.com/ubi8/ubi
```

Reply:
registry.access.redhat.com/ubi8/ubi works to pull

Reply:
@adambkaplan @xiuwang I've made the change.  If you give me your ok I would ask for peer review to get this merged.